.github/main.workflow

```diff
@@ -58,11 +59,7 @@ workflow "Auto-merge Crowdin PR" {
 action "Automerge PR" {
   uses = "actions/npm@master"
   args = "run automerge"
-  env = {
-    NODE_OPTIONS = "--max_old_space_size=4096"
-  }
   secrets = [
-    "CROWDIN_KEY",
```

Comment:
I'm unable to run this script without this `CROWDIN_KEY` env, it's unused but needed to pass `dotenv` checks.